### ISSUE TITLE
spark-wallet daemon doesn't release memory

### ISSUE THREAD

**Issue by m46nus** [shesek/spark-wallet#186]
Memory usage seems to be growing indefinitely. I'm running daemon on raspberry pi (raspbian - armv7l)

**Comment by m46nus** [comment]
I have version 0.2.17

**Comment by shesek** [comment]
You aren't using the `dev-server` mode by any chance, are you?

Can you tell if the memory usage grows even when no new connections are made to the web server?

**Comment by m46nus** [comment]
No, I'm not running in dev-server mode. I had it running for about a month and used wallet only couple times. So most probably it's consuming memory even if no new connections are made.
